Formula/vis.rb

```diff
@@ -40,10 +40,16 @@ def install
 
     system "./configure", "--prefix=#{libexec}"
     system "make", "install"
+
+    # Install the tools in bin under their default names since
+    # there's no clash with the system there and they expect
+    # to be executed from within the $PATH.
+    bin.install Dir[libexec/"bin/*"].reject { |f| f =~ /vis$/ }
```

Comment:
why not set --prefix=#{prefix} and then only move vis to libexec?

Reply:
Would need to rename the manpage then as well. Wasn't sure how much renaming you were happy with. If you're alright with that happy to flip it.

Reply:
I've made the chance but not force pushed, so there's a chance for you to decide if you hate it or not.